### ISSUE TITLE
Add Repeating Date module to BackdropCMS.org

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdropcms.org#942]
We have been trying with limited and sporatic success to always keep 3-4 Office Hours on the schedule manually. 

In the past, we avoided repeating date module to avoid a calendar that is simply full of Office Hours with everything else pushed off the page. However, after discussing the issue with @Wylbur and @yorkshire-pudding, we think it's time to try Repeating Dates. 

We recognize that there might be a few other adjustments we need to make for this to be effective. For example:

* Pinning "Featured" events to the top of the list OR
* Having office hours display in a different block from other events

We can experiment with this a bit, once we have the repeating dates module. 

@bugfolder - Can we install this module on the beta site for BackdropCMS.org and do some experimentation there?

**Comment by bugfolder** [comment]
> Can we install this module on the beta site for BackdropCMS.org and do some experimentation there?

Yes, I've added it to the beta.b.org branch. However, beta.b.org is currently having space problems. I'll ping @larsdesigns about it, will post an update here when the module is installed and beta is back up.

**Comment by bugfolder** [comment]
@stpaultim, the Repeating Dates module is set up on beta.b.org. Does it meet your needs? If so, we can put it on b.org, but if not, we'll remove it from beta.b.org.

**Comment by stpaultim** [comment]
I'm marking this as something to experiment with and test during our current Infrastructure week.

**Comment by bugfolder** [comment]
> I'm marking this as something to experiment with and test during our current Infrastructure week.

@stpaultim, did your "experiment with and test" on beta.b.org result in wanting this on b.org?
